### ISSUE TITLE
Add Support For Hardcore Bedrock

### ISSUE THREAD

**Issue by Austin-Higgins** [itzg/docker-minecraft-server#3127]
### Enhancement Type

Not sure

### Describe the enhancement

Latest branch does not have the environmental variable config for HARDCORE parameter. This is something I can help add if needed. I would really like to use this feature. :)

**Comment by itzg** [comment]
Yeah, that would be great if you could add a property mapping for that in https://github.com/itzg/docker-minecraft-bedrock-server/blob/master/property-definitions.json

**Comment by myxtrastufff** [comment]
I do not see the hardcore mode added to the link above. Has this feature been added? I am unable to use hardcore mode.

**Comment by itzg** [comment]
I just realized the title says "bedrock" but this repo is for Java edition. `HARDCORE` is already a mapped server property

https://github.com/itzg/docker-minecraft-server/blob/fd83920383b697ae59c3d35093e515e11e9487f0/files/property-definitions.json#L22

**Comment by myxtrastufff** [comment]
I see what you mean. My apologizes, I found this issue through google and did not notice that it was Java.  Is it possible to add hardcore for bedrock?

**Comment by itzg** [comment]
Bedrock doesn't seem to have that property:

https://minecraft.fandom.com/wiki/Server.properties#Option_keys
